lib.php

```diff
@@ -43,9 +44,6 @@ function accredible_add_instance($post) {
 
     $post->instance = isset($post->instance) ? $post->instance : null;
 
-    $localgroups = new groups();
-    $groupid = $localgroups->sync_group_with($course, $post->instance, $post->groupid);
```

Comment:
`$groupid` is still being called on line [57](https://github.com/accredible/moodle-mod_accredible/pull/56/files#diff-8f5ff72ef1cf59199cfc4bbf16f2475d09e448601242f0d2c51bb94bd4aa1395R57)
This leads to `Undefined variable: groupid` error.

Reply:
You were right, I have fixed